src/table/src/TableRow.js

```diff
@@ -58,7 +58,6 @@ export default class TableRow extends PureComponent {
     if (this.props.isSelectable) {
       if (e.key === 'Enter' || e.key === ' ') {
         this.props.onSelect()
-        e.preventDefault()
```

Comment:
Nice! Should we find another way to allow consumers to _opt-in_ to preventing scroll of the page upon these key presses?

Reply:
@mshwery Thanks for the feedback and additional context on the original implementation here. Appreciate it!

As an alternative to additional configuration for opt-in behavior, it occurred to me that we could instead continue to invoke `preventDefault` but first wrap the function call within a conditional that would first check if the event target is in fact the TableRow itself.

This way, if consumers of 🌲 decide to nest additional UI within an `isSelectable` `TableRow` that can take focus, or accept user input, the browser can and will scroll and direct focus to these child UI elements when the event originates there. This is good, and helps with web accessibility standards.

For an `isSelectable` `TableRow` that contains text or other children that do not provide a target to fire a keyPress event, `preventDefault` will not be called and browser scrolling will not occur when selecting a row. This scenario is captured in the existing [storybook example here](https://github.com/segmentio/evergreen/blob/master/src/table/stories/index.stories.js#L36-L42)